backend/fms_core/serializers.py

```diff
@@ -784,11 +784,15 @@ def get_steps_order(self, instance):
 
 class ReferenceGenomeSerializer(serializers.ModelSerializer):
     editable = serializers.SerializerMethodField()
+    taxon_id = serializers.SerializerMethodField()
     class Meta:
         model = ReferenceGenome
-        fields = ("id", "assembly_name", "synonym", "genbank_id", "refseq_id", "taxon_id", "size", "editable")
-    def get_editable(selft, obj):
-        return not Individual.objects.filter(reference_genome_id=obj.id).exists()
+        fields = "__all__"
+        extra_fields = ("taxon_id", "editable")
+    def get_editable(self, obj):
+        return not Individual.objects.filter(reference_genome_id = obj.id).exists()
```

Comment:
This would be cleaner to create a can_edit_reference_genome service in ../services/reference_genome.py to make sure we always have the same version of this test. This would include a small unit test also for the service(.../tests/test_services/reference_genome_test.py).